### PR TITLE
Renamed Orbit to Location schema

### PR DESCRIPTION
**Pull request by Vidushi-GitHub**
As discussed, it represents location of spacecraft rather than orbit. https://github.com/nasa-gcn/gcn-schema/issues/10 is still incomplete. @shb46

### CODE REVIEW

**Comment by lpsinger** [comment]
This would be easily confused with Localization.

What this data structure really describes is the local geomagnetic field.

**Comment by Vidushi-GitHub** [comment]
I agree. What about SpacecraftLocation?

**Comment by lpsinger** [comment]
Well, it's not the spacecraft location either. It's a random mix of spacecraft location and geomagnetic parameters. I don't think that they belong together.

**Comment by jracusin** [comment]
What about GeoLocation, which is where Boyan started?

File: gcn/notices/core/Location.schema.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "$id": "https://gcn.nasa.gov/schema/gcn/notices/core/GeoLocBase.schema.json",
+  "$id": "https://gcn.nasa.gov/schema/gcn/notices/core/Location.schema.json",
   "$schema": "https://json-schema.org/draft/2020-12/schema",
   "title": "Spacecraft Geolocation and McIlwain L",
   "description": "Enables GCN providers to include spacecraft geolocation and McIlwain L-Parameter.",
```
Comment (jracusin) [inline gcn/notices/core/Location.schema.json:5]:
```suggestion
  "description": "Enables GCN providers to include observatory geolocation and McIlwain L-Parameter.",
```

**Comment by lpsinger** [comment]
> What about GeoLocation? This is where Boyan started.

No, that doesn’t accurately describe this data structure either.

I see the use of a data structure that captures the local geomagnetic field AND a separate data structure that captures the full kinematic state of the spacecraft. However, I see no reason why these should be part of the same data structure.

Also, we should seek domain expertise (space weather and orbital dynamics) in the design of both data structures.

**Comment by jracusin** [comment]
@shb46, does Glowbug calculate the McIlwain-L onboard?  Does this value need to be reported?  GBM does not report it in their GCN Classic Notices.  We also found that the GBM Data Tools has a function to calculate McIlwain-L based upon latitude and longitude. https://fermi.gsfc.nasa.gov/ssc/data/analysis/rmfit/gbm_data_tools/gdt-docs/api/api-utils.html

**Comment by lpsinger** [comment]
It's highly unlikely that Glowbug calculates that onboard.

I'm not arguing against a data structure for geomagnetic field parameters. However, I strongly urge you to look do a review of different parametrization and consult some space weather experts on the design.

**Comment by Vidushi-GitHub** [comment]
As no instrument sends notices with spacecraft location and we don't have design idea, thus closing it. We can open a new PR if such requirement arises in future.

**Pull request closed without merging**